src/main/cpp/mechanisms/DragonTale/DragonTale.cpp

```diff
@@ -1084,6 +1084,12 @@ void DragonTale::RunCommonTasks()
 {
 	// This function is called once per loop before the current state Run()
 	SetSensorFailSafe();
+
+	if ((m_ElevatorLeader->GetReverseLimit().GetValue() == ReverseLimitValue::ClosedToGround) && (GetElevatorHeight() > 1_in))
+	{
```

Comment:
This logic doesn't work, the problem is that the Cancoder thinks it is at 0 in cause it slipped, but in actuality it is higher. I think we need to add an Units::math::abs around the GetElevatorHeight()